### ISSUE TITLE
Unable to install PostGIS:  could not open extension control file 

### ISSUE THREAD

**Issue by booboothefool** [PostgresApp/PostgresApp#403]
I installed PostgresQL via: http://postgresapp.com

I then ran the following commands:
```
    psql -U postgres
    Password for user postgres:
    psql (9.6.4, server 9.6.3)
    Type "help" for help.
    
    postgres=# \c dvdrental
    psql (9.6.4, server 9.6.3)
    You are now connected to database "dvdrental" as user "postgres".
    dvdrental=# CREATE EXTENSION postgis;
    ERROR:  could not open extension control file "/Library/PostgreSQL/9.6/share/postgresql/extension/postgis.control": No such file or directory
    dvdrental=#
```
What am I missing?

**Comment by jakob** [comment]
You are probably connected to a different server than the one from Postgres.app

Type the following command in psql:
`show data_directory;`

It should print something like this:
` /Users/jakob/Library/Application Support/Postgres/var-9.6`

If it prints something different, you have a different PostgreSQL installation running, which probably doesn't include PostGIS

**Comment by booboothefool** [comment]
@jakob Thank you for the lightning fast response time!

So this isn't Postgres.app?
```
which psql
/Applications/Postgres.app/Contents/Versions/9.6/bin/psql
psql -U postgres
Password for user postgres:
psql (9.6.4, server 9.6.3)
Type "help" for help.

postgres=# show data_directory;
        data_directory
------------------------------
 /Library/PostgreSQL/9.6/data
(1 row)
```
I don't remember installing another version of PostgresQL? I recall using Postgres.app because it was the only one I could get to install properly (dragging into Applications)!

**Comment by jakob** [comment]
It looks like you are using psql (the client) from Postgres.app (Version 9.6.4), but the server is running a different version (9.6.3). I think the EnterpriseDB installer might use  /Library/PostgreSQL as a data directory, try removing that before starting Postgres.app.

Instructions: https://postgresapp.com/documentation/remove.html

After removing, restart your computer to make sure no PostgreSQL processes are still running.

**Comment by booboothefool** [comment]
@jakob You were right!

Removing EnterpriseDB like you linked in the instructions worked.

I am not sure how this happened exactly but I have a hunch that when I installed pgadmin 4, it installed it's own version of PostgresQL, and connected to that instead. This whole time I thought I was connecting to Postgres.app server. 

Something I did when I installed Postgres.app was I deleted the default server "PostgresSQL 9.6". I believe I installed pgadmin 4 after that, which spun up it's own server? Then Postgres.app would always read "port already in use". I thought this meant that Postgres.app was already running, but this is not the case as I know see the green check marking with "Running" for the first time.
